layers/+distributions/spacemacs-bootstrap/packages.el

```diff
@@ -155,7 +155,10 @@
     (advice-add 'evil-refresh-cursor :around #'spacemacs/not-in-pdf-view-mode))
 
   (when vim-style-enable-undo-region
-    (define-key evil-visual-state-map (kbd "u") 'undo))
+    (define-key evil-visual-state-map (kbd "u")
+                (if (eq dotspacemacs-undo-system 'undo-fu)
+                    'undo
+                  'evil-undo)))
```

Comment:
Would it be correct to just bind this to `evil-undo` always?  I believe that should respect whatever the current `evil-undo-system` is, right?

Reply:
Oh, I misunderstood.  `undo` is being used to specifically avoid going through undo-fu?

Did you have `undo-fu-allow-undo-in-region` non-nil?

> I think we could also just remove undo-fu at some point, it does not seem to have enough benefits.

I agree.  I use undo-fu personally but I don't think I really take advantage of it, it mostly just gets in the way sometimes.